vta/microsites
# Large Format Printing #24

|#424 https://projects.vta.org/projects/copy-center/work_packages/424

This PR re-designs the **Large Format Printing** form similar to the new design of **Standard-Size Printing**. Updates were also made to **Standard-Size Printing** per Copy Center team's feedback.

**Testing:**
Go through both **Standard-Size Printing** and **Large Format Printing** forms.
1. Check smaller screen sizes on each page of both forms.
2. Select a date before 3-business days for minimum date validation.
3. In **Standard-Size Printing** on Page 2, select more than 1 choice for **Paper Size** to see if an additional text input renders. The additional field should be required.
4. In **Standard-Size Printing** on Page 3, if you select levels 2 or 3, select from one of the 3 columns/categories to see if the remaining 2 are disabled. If you unselect the input from that column completely, then all 3 categories should become active again.

**Requirements:**

**Large Format Printing Form**
- [x] Re-use the same background colors, font colors, and font-sizes from **Standard-Size Printing** & **Business Card Printing** forms.
- [x] Tabs should have navigation capabilities with form validation included.
- [x] The form should be mobile responsive.

**Standard-Size Printing Form (Fixes: User Test Feedback):**
- [x] Change CSS selectors so that similar styles are applied to both forms.
- [x] Move **Other Finishing/Binding Options** to the last page of the form.
- [x] Add additional instructions to **Staple/Hole Punch** option under **Other Finishing/Binding Options.**
- [x] On Page 2, if a user chooses more than 1 paper size option, display another text field to ask specific the reason for different paper sizes.
- [x] On Page 3, include instructions to indicate that only 1 of the 3 categories can be selected under **Other Finishing/Binding Options** for levels 2 & 3. Ensure this with custom form controls.

**Both Forms:**
- [x] Automatically set **Due Date** for _both_ forms 3 business days from today.
- [x] Expand the form to cover more width in mobile view
- [x] Ensure that input containers **(Proof Required & Print Type)** display choices vertically on smaller screens.

**Mockups:**
[Original Prototype](https://xd.adobe.com/view/a116b006-50f7-4f67-456f-b4651534bca6-d727/)
